CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
Can you change to say `## Main` on line 1.

Reply:
Done